app/stac_api/collection_spatial_extent.py

```diff
@@ -1,163 +1,2 @@
-                        )
-                    else:
-                        self.extent_geometry = geometry_updated_item
-                    logger.info(
-                        'Collection extent_geometry updated to %s in %ss, after item update',
-                        self.extent_geometry.extent,
-                        time.time() - start,
-                        extra={
-                            'collection': self.name, 'item': item.name, 'trigger': 'item-update'
-                        },
-                    )
-                updated |= True
-
-            # delete, we need to iterate trough the items
-            if trigger == 'delete':
-                logger.warning(
-                    'Updating collections extent_geometry with removal of item geometry %s. '
-                    'We need to loop over all items of the collection, this may take a while !',
-                    GEOSGeometry(geometry).extent,
-                    extra={
-                        'collection': self.name, 'item': item.name, 'trigger': 'item-delete'
-                    },
-                )
-                start = time.time()
-                bbox_other_items = type(item).objects.filter(collection_id=self.pk).exclude(
-                    id=item.pk
-                ).only('geometry', 'collection').aggregate(Extent('geometry'))
-                if bool(bbox_other_items['geometry__extent']):
-                    self.extent_geometry = GEOSGeometry(
-                        Polygon.from_bbox(bbox_other_items['geometry__extent'])
-                    )
-                else:
-                    self.extent_geometry = None
-                logger.info(
-                    'Collection extent_geometry updated to %s in %ss, after item deletion',
-                    self.extent_geometry.extent if self.extent_geometry else None,
-                    time.time() - start,
-                    extra={
-                        'collection': self.name, 'item': item.name, 'trigger': 'item-delete'
-                    },
-                )
-                updated |= True
-        except GEOSException as error:
-            logger.error(
-                'Failed to update spatial extend in collection %s with item %s, trigger=%s, '
-                'current-extent=%s, new-geometry=%s, old-geometry=%s: %s',
-                self.name,
-                item.name,
-                trigger,
-                self.extent_geometry,
-                GEOSGeometry(geometry).extent,
-                GEOSGeometry(original_geometry).extent,
-                error,
-                extra={
-                    'collection': self.name, 'item': item.name, 'trigger': f'item-{trigger}'
-                },
-            )
-            raise GEOSException(
-                f'Failed to update spatial extend in colletion {self.name} with item '
-                f'{item.name}: {error}'
-            ) from error
-        return updated
+    pass
```

Comment:
why do we have to keep this empty class?

Reply:
Because of old migration file that reference this class. I would need to check with @boecklic and/or search on Internet what is the best approach for such migration. But for the moment I think we are fine with an empty class.